OneCommunityGlobal/HighestGoodNetworkApp
# Masasa - added indentation to subpermissions in modal

# Description
Indentation necessary for subpermissions in the permissions modal component
<img width="414" alt="Screenshot 2023-08-22 at 1 26 17 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/4a821b9b-3120-42c1-9bbc-bbf8d150ce33">


## Related PRS (if any):
No related PRs
…

## Main changes explained:
- created a condition in UserPermissionsPopUp.jsx to differentiate between main permissions and sub_permissions and then added the necessary formatting

…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user
4. go to other links→ Permissions Management→ Manage User permissions
5. verify that the main subperimissions have been indented

## Screenshots or videos of changes:


<img width="664" alt="Screenshot 2023-08-24 at 3 30 47 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/4c9671e2-aac4-42c8-9761-b5e907394e26">
